src/components/cutscenes/IntroCutscene.ts

```diff
@@ -33,7 +33,7 @@ export class IntroCutscene<TGameStartr extends FullScreenPokemon> extends Genera
         this.gameStarter.things.add(oak);
         this.gameStarter.physics.setMidX(oak, this.gameStarter.mapScreener.middleX | 0);
         this.gameStarter.physics.setMidY(oak, this.gameStarter.mapScreener.middleY | 0);
-
+        
```

Comment:
The build is failing because of this line. 
https://travis-ci.org/FullScreenShenanigans/FullScreenPokemon/builds/332396272?utm_source=github_status&utm_medium=notification

You can check whether everything is ok by running `npm run verify` locally. For this particular failure, the `npm run src:tslint` task is failing because this is violating TSLint's [no-trailing-whitespace](https://palantir.github.io/tslint/rules/no-trailing-whitespace/) rule.

Reply:
I fixed the line and added a TSLint extension to my VS Code.